### PR TITLE
create post page backend

### PR DESCRIPTION
**Pull request by AlenaGavriluk**
On feed page: added max height to card. If there is no cover don't show cover. Don't show images inside the text. Use html or markdown mapping.
On create post page: logic of saving images and saving posts. 
On header: position fixed and z-index. It needs for using header with another pages.
On profile sidebar: fix styles and change one wrong svg
Added migration for two new fields in post table: markdown : boolean, coverImage : string (url)

### CODE REVIEW

**Comment by quipex-binary-studio** [comment]
Resolve conflicts

File: backend/core/app/src/main/java/com/mindbridge/core/domains/post/PostService.java [hunk 1]
```diff
@@ -46,6 +59,14 @@ public PostDetailsDto getPostById(UUID id) {
 		return post;
 	}
 
+	public void savePost(CreatePostDto post) {
+		System.out.println(post);
```
Comment (Cooplix) [inline backend/core/app/src/main/java/com/mindbridge/core/domains/post/PostService.java:63]: Use logger
https://stackoverflow.com/questions/31869391/what-is-the-difference-between-java-logger-and-system-out-println

Comment (AlenaGavriluk) [reply backend/core/app/src/main/java/com/mindbridge/core/domains/post/PostService.java]: > Use logger
> https://stackoverflow.com/questions/31869391/what-is-the-difference-between-java-logger-and-system-out-println

I used logger in another files and in these place it's not necessary so I just delete the row

File: frontend/src/components/PostPreview/index.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+interface IPreviewFormProps {
+  form: IForm;
+  modes: IModes;
+}
+
+const PostPreview: React.FC<IPreviewFormProps> = ({ form, modes }) => (
+  <div className={styles.post_preview_container}>
+    {form.coverImage
+      ? <img height="" width="" className={styles.coverImage} src={form.coverImage.url} alt="cover" />
+      : null}
```
Comment (Quipex) [inline frontend/src/components/PostPreview/index.tsx:15]: form.coverImage && <img...

File: frontend/src/screens/CreatePost/containers/CreatePostPage/index.tsx [hunk 3]
```diff
@@ -1,11 +1,10 @@
-import React, { useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
 import styles from './styles.module.scss';
 import classNames from 'classnames';
 import { connect } from 'react-redux';
+import Header from '@root/components/Header';
```
Comment (Quipex) [inline frontend/src/screens/CreatePost/containers/CreatePostPage/index.tsx:5]: defined but never used?

**Review by Quipex — APPROVED** [review]
fine, except some redundant imports

**Pull request merged**